projects/deca/README.md

```diff
@@ -302,19 +302,13 @@ they specify a GHC version. These snapshots are curated by the community and
 FP Complete and can be found on [stackage.org](https://www.stackage.org/).
 
 ```yaml
-resolver: lts-18.27
-
-extra-deps:
-  - clash-prelude-1.6.3
-  - clash-lib-1.6.3
-  - clash-ghc-1.6.3
+resolver: lts-19.32
 ```
 
-This project uses [lts-18.27](https://www.stackage.org/lts-18.27), which
-includes Clash 1.4.7. We've added the extra-deps section to make sure Stack
-fetches the latest version of Clash, 1.6.3, instead. The point of this exercise
-is to make reproducible builds. Or in other words, if a `stack build` works
-now, it will work in 10 years too.
+This project uses [lts-19.32](https://www.stackage.org/lts-19.32), which
+includes Clash 1.6.4. Snapshots tightly couple GHC and package versions. On
+the one hand, this makes for a cohesive set of packages. On the other, it
+guarantees that if a `stack build` works now, it will work in 10 years too.
```

Comment:
I feel both
* cohesive set of packages
* guaranteed it will work in 10 years

are advantages. But "on the one hand ... on the other" suggests the two aspects have a stark contrast. I think it's not really the correct idiom here.

Reply:
Yeah that's fair, I'll rephrase and ask for another review.

Reply:
How do you feel about the current phrasing?

Reply:
:+1: